doctrine/reverse_engineering.rst

```diff
@@ -85,6 +85,24 @@ run:
     // generates getter/setter methods
     $ php bin/console make:entity --regenerate App
 
+If you generated XML or YAML metadata you have also to change the ``config/packages/doctrine.yaml``
+file like this (the example is for YAML):
+
+.. code-block:: yaml
+
+
+    doctrine:
+        ...
```

Comment:
this is invalid yaml, please use `# ...`

Reply:
Hi @OskarStark, do you mean something like this ?
```
    doctrine:
        ###
        orm:
            ###
            mappings:
                App:
                    is_bundle: false
                    type: yml # Previously was 'annotation'
                    dir: '%kernel.project_dir%/config/doctrine' # Previously was '%kernel.project_dir%/src/Entity'
                    prefix: 'App\Entity'
                    alias: App
```
If not how i can shorten the above example ?

Reply:
Forget my last comment, it should be:
```
    doctrine:
        # ...
        orm:
            # ...
            mappings:
                App:
                    is_bundle: false
                    type: yml # Previously was 'annotation'
                    dir: '%kernel.project_dir%/config/doctrine' # Previously was '%kernel.project_dir%/src/Entity'
                    prefix: 'App\Entity'
                    alias: App
```